<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorklistServerSettingsDialog</class>
 <widget class="QDialog" name="WorklistServerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIS Server</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>RIS System</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ipAddressLbl">
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ipAddresslineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLbl">
        <property name="text">
         <string>Remote Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="AETitleLbl">
        <property name="text">
         <string>AE Title</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="AETitleLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
